styles/carousel.css

```diff
@@ -0,0 +1,273 @@
+.cards-container {
+    display: flex;
```

Comment:
Is this a component, if yes it should be within the blocks folder

Reply:
Changes done.